packages/helpers/src/exec-command-wrapper.ts

```diff
@@ -0,0 +1,123 @@
+import * as core from '@actions/core'
+import * as exec from '@actions/exec'
+
+export class Command {
```

Comment:
It is better to alter the name to something like `CommandHelper` because there's already `Command` module within toolkit that is used under the hood of `exec` module.